pkg/vm/vm.go

```diff
@@ -100,6 +103,8 @@ func NewVirtualMachine(
 		Trace:        trace,
 		config:       config,
 		instructions: make(map[uint64]*a.Instruction),
+		RcLimitsMin:  math.MaxUint64,
```

Comment:
again why is max 0 and min maxuint64?

Reply:
Those are just initial values for comparison with the actula values, which will likely be more than 0 for `RcLimitsMax` and less than `MaxUint64` for `RcLimitsMin`